JAVA/note-app/README.md

```diff
@@ -0,0 +1,72 @@
+# Note App
+
+<p align="center">
+<img src="blob/noteapp.png">
+</p>
```

Comment:
remove this image in place of that add 4 to 5 lines Description of your project.

Reply:
@heet-kumar So I make a video of the code walkthrough and remove the screenshot. That's all right?

Reply:
@heet-kumar I've done the requested changes. And the setting up of this project is mentioned clearly in my readme file.

Reply:
@rajdip-b ok. 👍

Reply:
> @heet-kumar So I make a video of setting up the project locally, the code walkthrough and remove the screenshot. That's all right?

@rajdip-b Yes I want that.